app/javascript/packages/url/add-search-params.ts

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
This is just a lift-and-shift from the previous `index.ts`, apparently git/GitHub didn't pick it up as such.